db/models.py

```diff
@@ -1 +1,16 @@
 from django.db import models
+
+
+class Genre(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+
+    def __str__(self) -> str:
+        return f"Genre: {self.name}"
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self) -> str:
+        return f"Actor: {self.first_name} {self.last_name}"
```

Comment:
Do not include the model name in the `__str__` method in the future, it is present by default.